src/app/shared/shared.module.ts

```diff
@@ -80,6 +81,7 @@ import { AccessRolePipe } from './pipes/access-role.pipe';
     NgbDropdownModule,
     NgbPaginationModule,
     FontAwesomeModule,
+    ComponentsModule,
```

Comment:
```suggestion
```
Let's get rid of all ComponentsModule imports for now until they're actually needed.